jetbrains-gateway/main.tf

```diff
@@ -295,8 +297,8 @@ resource "coder_app" "gateway" {
     data.coder_workspace.me.name,
     "&owner=",
     data.coder_workspace_owner.me.name,
-    "&agent=",
-    var.agent_name,
+    #    "&agent=",
+    #    var.agent_name,
```

Comment:
```suggestion
```
I think it is ok to remove these lines. Even if someone specifies the input it will not be used.

Reply:
I did remove it, I just had uncommited changes, sorry!